src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -1272,38 +1139,12 @@ public function testSubmitMultipleExpandedObjectChoices()
         $this->assertNull($form[4]->getViewData());
     }
 
-    public function testSubmitMultipleExpandedNumericChoices()
```

Comment:
You're right. But in fact, it fails. I double checked with webmozart, it's to be removed